plugins/wordpress-rule-exclusions-before.conf

```diff
@@ -875,6 +878,8 @@ SecRule REQUEST_FILENAME "@rx /wp-admin/load-(?:scripts|styles)\.php$" \
     ctl:ruleRemoveById=921180,\
     ctl:ruleRemoveTargetById=920100;REQUEST_LINE,\
     ctl:ruleRemoveTargetById=920273;ARGS_NAMES:load[],\
+    ctl:ruleRemoveTargetById=932236;ARGS:load[chunk_0],\
+    ctl:ruleRemoveTargetById=932236;ARGS:load[chunk_1],\
```

Comment:
Also here. You ordered `ruleRemoveTargetById` based on rule ID but we used to order it by `variable,argument,rule`.

Reply:
is this better?

Reply:
@azurit?

Reply:
@azurit is everything here good?

Reply:
Sorry guys, all good!